### ISSUE TITLE
Gingerbread: flag watched applied only after back press

### ISSUE THREAD

**Issue by UweTrottmann** [UweTrottmann/SeriesGuide#249]
Go into a shows overview, try to flag the episode shown as watched.
Nothing happens until you press back to return to the show list.

Not yet confirmed with trakt disconnected.

Likely won't fix this legacy platform bug as it works fine on consecutive actions, this seems to happen only the first time.
